File: portfolio-BackEnd/src/main/java/com/portfolio/ag/Entity/Studies.java
package com.portfolio.ag.Entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Studies {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String studyName;
    private String studyInfo;
    
    //Constructors
    public Studies() {
    }

    public Studies(String studyName, String studyInfo) {
        this.studyName = studyName;
        this.studyInfo = studyInfo;
    }
    
    //Getters - setters
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getStudyName() {
        return studyName;
    }

    public void setStudyName(String studyName) {
        this.studyName = studyName;
    }

    public String getStudyInfo() {
        return studyInfo;
    }

    public void setStudyInfo(String studyInfo) {
        this.studyInfo = studyInfo;
    }
}

File: src/app/model/studies.ts
export class Studies{
    id?: number;
    studyName: string;
    studyInfo: string;
    studyDate: string;
    studyImg: string;

    constructor (studyName: string, studyInfo: string, studyDate: string, studyImg: string) {
        this.studyName = studyName;
        this.studyInfo = studyInfo;
        this.studyDate = studyDate;
        this.studyImg = studyImg;
    }
}